src/components/App.js

```diff
@@ -1,7 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Button from 'components/Button/Button'
+import Container from 'components/Container/Container'
+import Input from 'components/Input/Input'
+import List from 'components/List/List'
+import ListItem from 'components/ListItem/ListItem'
 
 const App = () => {
-  return <></>
+  const [items, setItems] = useState([])
```

Comment:
Me faltó un comentario previo, `items` es un nombre muy genérico y no explicita bien a qué hace referencia. Mejor hacerlo un poco más específico.

```suggestion
  const [todos, setTodos] = useState([])
```